stacks/single-server/launch.sh

```diff
@@ -1,16 +1,48 @@
 #!/bin/bash
 
+# OpenEMR Cloud Express launcher
+# usage: launch.sh -s 2 -b wip-feature -d
+#        -s: amount of swap to allocate, in gigabytes
+#        -b: repo branch to load instead of master
+#        -d: start in developer mode, force local dockers and open ports
```

Comment:
This flag might require more description. I'm not sure what 'force local dockers' means and you may want to specify if you're opening container or host ports.

Reply:
It's declining to use the openemr at the hub, and forcing the use of the one in the current branch of the repo instead, the one that just got checked out. It's also forcing the MySQL container to open 3306 to the host; usually this isn't done.

Reply:
Oh sorry. I meant that it might be useful to be more explicit about that in the help page there